contracts/infrastructure/storage/LimitStorage.sol

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2018  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.5.4;
```

Comment:
I've separated all infrastructure contracts that need to be compiled with solc 0.5 into their own `infrastructure_0.5` folder.

Reply:
What is `infrastructure_0.5` used for? Also, since you didn't specify a different `--buildDirectory` for `compile:infrastructure_0.5`, I think the artifacts that this command produces will just be overwritten by the ones produced by `compile:infrastructure_0.6`.

Reply:
This is just to separate the infrastructure contracts that can be compiled with solc 0.6 and the ones that are currently in prod and require 0.5. There is no overlapping between the 2 folders.